server/container_start.go

```diff
@@ -34,8 +39,20 @@ func (s *Server) StartContainer(ctx context.Context, req *pb.StartContainerReque
 		if err := s.ContainerStateToDisk(c); err != nil {
 			log.Warnf(ctx, "unable to write containers %s state to disk: %v", c.ID(), err)
 		}
+
+		if useHighPerformanceRuntimeHandler && shouldCPULoadBalancingBeDisabled(c.Annotations()) {
+			if err := setCPUSLoadBalancing(c, true, schedDomainDir); err != nil {
+				log.Warnf(ctx, "failed to set back the container %q CPUs load balancing to true: %v", c.ID(), err)
+			}
+		}
 	}()
 
+	if useHighPerformanceRuntimeHandler && shouldCPULoadBalancingBeDisabled(c.Annotations()) {
+		if err := setCPUSLoadBalancing(c, true, schedDomainDir); err != nil {
```

Comment:
this is still true but the error says false

Reply:
thanks, done